composer.json

```diff
@@ -17,31 +17,31 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
-        "ext-mongodb": "^1.1.5",
-        "facile-it/mongodb-bundle": "^0.6 || ^1.0.0",
-        "mongodb/mongodb": "^1.1.0",
+        "php": "^7.4 || ^8.1",
+        "ext-mongodb": "^1.6",
+        "facile-it/mongodb-bundle": "^1.6",
+        "mongodb/mongodb": "^1.5",
         "symfony/deprecation-contracts": "^2.1",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/messenger": "^4.4 || ^5.0 || ^6.0"
+        "symfony/framework-bundle": "^5.0 || ^6.0 || ^7.0",
+        "symfony/messenger": "^5.0 || ^6.0 || ^7.0"
```

Comment:
Are we dropping Symfony 4.4? If we do, we should bump up to 5.4, the other 5.x minors are not LTS.